server/services/offencesPageService.ts

```diff
@@ -186,11 +192,13 @@ export default class OffencesPageService {
     let courtName: string
     if (courtCase.agency && courtCase.agency.description) {
       courtName = courtCase.agency.description
-    } else if (courtCase.court && courtCase.court.description) {
+      return courtName
+    }
+    if (courtCase.court && courtCase.court.description) {
       courtName = courtCase.court.description
-    } else {
-      courtName = undefined
+      return courtName
     }
+    courtName = undefined
```

Comment:
Could get rid of the `let courtName` temporary variable and just return `courtCase.agency.description` etc in one line

Reply:
Thanks Jon, I'll raise another PR for that change later on.